src/ocamlorg_frontend/pages/install.eml

```diff
@@ -120,12 +120,11 @@ Layout.render
         </h2>
         <ol>
           <li>
-            <h3>Install the DkML Windows distribution</h3>
+            <h3>Install the opam package manager</h3>
               <p>
-                The installer sets up OCaml 4.14.0 and <a target="_blank" href="https://github.com/ocaml/opam">OCaml's package manager opam</a>,
-                with Git and the Visual Studio compiler.
+                OCaml's package manager opam's 2.2 version onwards supports Windows natively. It is the recmmended way to install OCaml on Windows machines.

```

Comment:
```suggestion
                OCaml's package manager, opam, supports Windows natively since version 2.2.0 and is the recommended way to install OCaml on Windows.
```